internal/gatewayapi/listener.go

```diff
@@ -175,6 +175,10 @@ func processAccessLog(envoyproxy *configv1a1.EnvoyProxy) *ir.AccessLog {
 		for _, sink := range accessLog.Sinks {
 			switch sink.Type {
```

Comment:
Do we need a default in switch to deal with the unsupported sink type? @zirain

Reply:
that will block by kube-apiserver?

Reply:
ya, I think that was already prevented from the schema validation by kube-apiserver, as `sink.Type` is actually an enum of `File` and `OpenTelemetry`.

Reply:
Right, thanks.